### PR TITLE
Add example certificate and autogenerate pretty versions

### PR DESCRIPTION
**Pull request by bwesterb**

### CODE REVIEW

File: examples/ML-DSA-65.crt.txt [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+d4d648a0d008b6af5e8cd31be69a9296d4f3fd25ed86f221e4b93f65f5929967
+533624b9235750c30707550b58536d109a7131c5a5bbe4a5715567c12534aec7
+660761eebb9fae2891c774589b80e566ad557ddef7367196b7227ea9870ef09d
+dfec79d6b9319a6879b5205d76bf7aba5acf33afb59d17fc54e68383d6be5a08
+e9b66da53dcde008bb294b8582bd132cdcc49959fdbc21e52721880c8ad0352c
+79f03a43bbd84c4cdfdc6c529005e1e7cd9a349a7168a35569ba5dea818968d5
+a91466bd6e64e20bf62417198afc4e81c28dd77ed4028232398b52fbde86bc84
+f475b9016710ce2aabc11a06b4dbac901ec16cf365ca3f2d53813948a693a0f9
+3e79c46ca5d5a6dca3d28ca50ad18bd13fca55059dd9b185f79f9c47196a4e81
+b2104bc460a051e02f2e8444f` }
```
Comment (jakemas) [inline examples/ML-DSA-65.crt.txt:111]: There seems to be a compatibility issue here with `der2ascii`:
```
Syntax error: line 111: encoding/hex: invalid byte: U+000A
```

File: examples/ML-DSA-65.crt
Comment (jakemas) [inline examples/ML-DSA-65.crt:1]: Was able to successfully generate this cert from PEM, extract the public key, and verify in AWS-LC. See `kMLDSA65Cert` in the gtest `TestMLDSA65`.

Added to AWS-LC:

https://github.com/aws/aws-lc/pull/1963/commits/c71bbac49160038ba8262b9f1996b43e197410ba#diff-e8a50b868e2b0f2f27afa3c9b1fbc844dce0cac6883ea9615882149e16eb4e35

**Review by jakemas — APPROVED** [review]

File: examples/ML-DSA-65.crt.txt (hunk 1 above)
Comment (bwesterb) [reply examples/ML-DSA-65.crt.txt:111]: It's because the long lines are wrapped using `fold`. Check out `examples/pretty`.

File: examples/ML-DSA-65.crt
Comment (bwesterb) [reply examples/ML-DSA-65.crt:1]: Great, thanks for testing!

**Pull request merged**